components/builder-originsrv/src/server/mod.rs

```diff
@@ -333,7 +329,6 @@ lazy_static! {
 pub struct ServerState {
     datastore: DataStore,
     sessions: Arc<Box<RwLock<HashSet<Session>>>>,
```

Comment:
Do we need the sessions set here either?  We may be able to get rid of this later (different PR)

Reply:
I wasn't sure - I figured I'd keep my scope of impact smaller on this PR and look into moving more session stuff over after this lands.

Reply:
yep agreed